packages/editor-ui/src/views/canvasHelpers.ts

```diff
@@ -49,7 +49,7 @@ export const DEFAULT_START_NODE = {
 };
 
 export const WELCOME_STICKY_NODE = {
-	name: "Note1",
+	name: "QuickstartNote",
```

Comment:
can you please use something a lot more unique? really want to avoid this conflicting with what a user might name their node... so maybe `__QUICKSTART_NOTE__`... in fact you should make this its own constant so you can import it when checking for telemetry events

Reply:
Ok, will update.